ppr-ui/src/composables/useInputRules.ts

```diff
@@ -42,6 +48,22 @@ export const useInputRules = () => {
     ]
   }
 
+  // Check if string starts with any of the search values
+  const startsWith = (errorMessage: string, ...searchValues: string[]): Array<Function> => {
```

Comment:
Good use of Rest parameter 👍

Reply:
I refactored this to be consistent and put the error message as the last parameter. Changed from rest param to an array. Looks cleaner.